src/RwaLiquidationOracle.sol

```diff
@@ -111,14 +111,14 @@ contract RwaLiquidationOracle {
     }
     // --- write-off ---
     function cull(bytes32 ilk, address urn) external auth {
-        require(ilks[ilk].pip != address(0));
-        require(add(ilks[ilk].toc, ilks[ilk].tau) >= block.timestamp);
+        require(ilks[ilk].pip != address(0), "RwaOracle/unknown-ilk");
+        require(block.timestamp >= add(ilks[ilk].toc, ilks[ilk].tau), "RwaOracle/early-cull");
```

Comment:
@livnev this is what we want right?

Reply:
The new version is correct, yes. `cull` should only be callable once the remediation period has elapsed. Hopefully the revert message also documents the intent here a bit.